api/nakadi-event-bus-api.yaml

```diff
@@ -1083,6 +1131,35 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
```

Comment:
Maybe I'm missing a fundamental part of the concept here, but how exactly do subscriptions work? I was expecting to specify a URI in here which then receives events, being pushed by Nakadi. Am I wrong?